### ISSUE TITLE
BS 8000-0 fetching error

### ISSUE THREAD

**Issue by Intelligent2013** [relaton/relaton-bsi#13]
Source issue: https://github.com/metanorma/metanorma-bsi/issues/115

The code:
```ruby
require 'relaton_bsi'
item = RelatonBsi::BsiBibliography.get("BS 8000-0")
```
raises the error:
```
[relaton-bsi] ("BS 8000-0") fetching...
Traceback (most recent call last):
        8: from pas_test.rb:8:in `<main>'
        7: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/bsi_bibliography.rb:38:in `get'
        6: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/bsi_bibliography.rb:114:in `bib_get1'
        5: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/bsi_bibliography.rb:75:in `search_filter'
        4: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/bsi_bibliography.rb:20:in `search'
        3: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/bsi_bibliography.rb:20:in `new'
        2: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/hit_collection.rb:22:in `initialize'
        1: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/hit_collection.rb:45:in `hits'
C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/relaton-bsi-1.10.1/lib/relaton_bsi/hit_collection.rb:45:in `sort_by': comparison of NilClass with String failed (ArgumentError)
```
https://shop.bsigroup.com/search?productType=standard&query=BS%208000-0 returns the document:
![image](https://user-images.githubusercontent.com/5559926/154972900-4f7a8b12-5cda-436e-8706-cd9dca2c8348.png)

**Comment by andrew2net** [comment]
fixed in -v 1.10.2

**Comment by Intelligent2013** [comment]
@andrew2net thanks!
